Calculate average value in timerlat-hist summary with two-digit 
precision to avoid displaying too optimitic results. 

Signed-off-by: Andreas Ziegler <br015@umbiko.net>
---
diff -Naur a/tools/tracing/rtla/src/timerlat_hist.c b/tools/tracing/rtla/src/timerlat_hist.c
--- a/tools/tracing/rtla/src/timerlat_hist.c	2023-02-14 19:11:56.000000000 +0100
+++ b/tools/tracing/rtla/src/timerlat_hist.c	2023-02-22 10:41:15.478864281 +0100
@@ -349,16 +349,16 @@
 
 		if (!params->no_irq) {
 			if (data->hist[cpu].irq_count)
-				trace_seq_printf(trace->seq, "%9llu ",
-						 data->hist[cpu].sum_irq / data->hist[cpu].irq_count);
+				trace_seq_printf(trace->seq, "%9.2f ",
+						 ((double) data->hist[cpu].sum_irq) / data->hist[cpu].irq_count);
 			else
 				trace_seq_printf(trace->seq, "        - ");
 		}
 
 		if (!params->no_thread) {
 			if (data->hist[cpu].thread_count)
-				trace_seq_printf(trace->seq, "%9llu ",
-						 data->hist[cpu].sum_thread / data->hist[cpu].thread_count);
+				trace_seq_printf(trace->seq, "%9.2f ",
+						 ((double) data->hist[cpu].sum_thread) / data->hist[cpu].thread_count);
 			else
 				trace_seq_printf(trace->seq, "        - ");
 		}
